src/packet_distribution.cpp

```diff
@@ -161,10 +163,10 @@ void AOApplication::server_packet_received(AOPacket *p_packet)
       w_courtroom->append_server_chatmessage(f_contents.at(0),
                                              f_contents.at(1));
   }
-  // FIXME Should the FL packet acknowledgement removed?
   else if (header == "FL")
   {
-    // The packet is acknowledged but is of no use to the client
+    if (f_packet.contains("ackMS", Qt::CaseInsensitive))
+      ackMS_enabled = true;
```

Comment:
If we keep it in for the sake of making the client still somewhat compatible with the current server software, tag it with a TODO/NOTE to remove it in the future so that it's much easier to remove without having to scratch your head what needs to be removed anymore.